### DISCUSSION TITLE
Is there a way to hide line numbers?

### DISCUSSION THREAD

**Discussion by Yevgnen** [zed-industries/zed#6493] [Feature Requests]
Hi, 

Coming from Emacs, is there a way to hide line number? The `line, col` on the status bar is enough for me :-)

**Comment by JosephTLyons** [comment]
We don't have a setting for this currently.

**Comment by JosephTLyons** [comment]
We have a setting for this now:

```json
"gutter": {
  "line_numbers": false
}
```
